pkg/test/framework/components/apps/kube.go

```diff
@@ -664,24 +663,36 @@ func (a *kubeApp) Call(e AppEndpoint, opts AppCallOptions) ([]*echo.ParsedRespon
 		return nil, err
 	}
 
+	return resp, nil
+}
+
+func (a *kubeApp) ValidatedCall(e AppEndpoint, opts AppCallOptions) ([]*echo.ParsedResponse, error) {
```

Comment:
Missing checking for `resp[0].Port` from the previous `Call` function. @howardjohn